CosmosApiMessageStatusChangeFeed/function.json

```diff
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "function",
+      "type": "cosmosDBTrigger",
+      "name": "cosmosApiMessageStatusTrigger",
+      "direction": "in",
+      "connectionStringSetting": "COSMOS_API_CONNECTION_STRING",
+      "databaseName": "%COSMOSDB_NAME%",
+      "collectionName": "message-status",
+      "leaseCollectionName": "pdnd-leases-message-status-001",
```

Comment:
Could be useful to use the leaseCollectionPrefix to centralize the pdnd lease in a unique collection?

Reply:
Done in [91f07dc](https://github.com/pagopa/io-functions-elt/pull/35/commits/91f07dc867a84f939081af278863535ddc8841d0)